### PR TITLE
chore: added standards and license

### PR DESCRIPTION
**Pull request by antoniomuso**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,13 +14,15 @@
...
     "type": "git",
-    "url": "https://github.com/loblaw-sre/backstage-plugin-gitlab"
+    "url": "https://github.com/immobiliare/backstage-plugin-gitlab"
   },
   "keywords": [
     "backstage",
     "gitlab",
     "ci/cd",
-    "loblaw"
+    "loblaw",
```
Comment (JellyBellyDev) [inline package.json:23]: remove `loblaw`?

File: .github/workflows/test.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        steps:
+            - uses: actions/checkout@v3
+              with:
+                  token: ${{ secrets.GITHUB_TOKEN }}
+            - uses: actions/setup-node@v3
+              with:
+                  node-version: ${{ matrix.node-version }}
+
+            - name: Install dependencies
+              run: npm it
```
Comment (JellyBellyDev) [inline .github/workflows/test.yml:30]: `it`?

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+## 🚨 Proposed changes
```
Comment (JellyBellyDev) [inline .github/pull_request_template.md:1]: we can add also issue template?

File: NOTICE [hunk 4]
```diff
@@ -0,0 +1,5 @@
+Immobiliare Backstage Gitlab Plugin
```
Comment (JellyBellyDev) [inline NOTICE:1]: `GitLab`

File: LICENSE [hunk 5]
```diff
@@ -186,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2021 Sathish Kumar
+   Copyright [yyyy] [name of copyright owner]
```
Comment (JellyBellyDev) [inline LICENSE:190]: `2022 Immobiliare Labs` no?

File: dev/index.tsx [hunk 6]
```diff
@@ -1,3 +1,38 @@
+/**
+Copyright [yyyy] [name of copyright owner]
```
Comment (JellyBellyDev) [inline dev/index.tsx:2]: `2022 Immobiliare Labs` no?

**Review by JellyBellyDev — CHANGES_REQUESTED** [review]

File: .github/workflows/test.yml (hunk 2 above)
Comment (antoniomuso) [reply .github/workflows/test.yml:30]: install and test

File: package.json (hunk 1 above)
Comment (antoniomuso) [reply package.json:23]: ok

File: .versionrc.json [hunk 7]
```diff
@@ -0,0 +1,49 @@
+{
+    "tagPrefix": "v",
+    "name": "conventionalcommits",
+    "owner": "immobiliare",
+    "repository": "dats",
```
Comment (JellyBellyDev) [inline .versionrc.json:5]: shot!

File: LICENSE (hunk 5 above)
Comment (JellyBellyDev) [reply LICENSE:190]: 👍

File: .versionrc.json (hunk 7 above)
Comment (antoniomuso) [reply .versionrc.json:5]: you are a sniper!

File: dev/index.tsx (hunk 6 above)
Comment (antoniomuso) [reply dev/index.tsx]: For now, I removed the license for every file.

**Review by dnlup — COMMENTED** [review]
Are we planning to use `npm` or `yarn`? I think I see a mix of both between the CI jobs and the docs

File: LICENSE (hunk 5 above)
Comment (dnlup) [reply LICENSE:190]: > No this is an error, this LICENSE must not be changed. It should remain unaltered.

Where are we adding our license then? I am not sure about this 🤔

Comment (antoniomuso) [reply LICENSE:190]: In the NOTICE File

**Comment by antoniomuso** [comment]
Backstage uses yarn then I think we will use yarn.

File: .czrc [hunk 8]
```diff
@@ -0,0 +1,3 @@
+{
+  "path": "./node_modules/cz-conventional-changelog"
+}
```
Comment (JellyBellyDev) [inline .czrc:3]: add new line at end of file! :)

File: LICENSE (hunk 5 above)
Comment (JellyBellyDev) [reply LICENSE:190]: > It should remain unaltered.

ok, revert it! ;)

File: .versionrc.json [hunk 9]
```diff
@@ -0,0 +1,49 @@
+{
+    "tagPrefix": "v",
```
Comment (JellyBellyDev) [inline .versionrc.json:2]: `    "version": "0.1.24",` where is the `v`?  :)

File: .github/pull_request_template.md (hunk 3 above)
Comment (antoniomuso) [reply .github/pull_request_template.md:1]: yes, we can. But I will do it in another PR.

File: .versionrc.json (hunk 9 above)
Comment (antoniomuso) [reply .versionrc.json:2]: `"version": "0.1.24"` is the package json version; it is without v, and the `v` is added during tag creation.

Comment (JellyBellyDev) [reply .versionrc.json:2]: 👍

**Review by simonecorsi — APPROVED** [review]

File: README.md [hunk 10]
```diff
@@ -1,30 +1,12 @@
...
+![license](https://img.shields.io/github/license/immobiliare/backstage-plugin-gitlab)
 
+# Backstage GitLab Plugin
 
-Website: [https://gitlab.com/](https://gitlab.com/)
-
-<img src="https://raw.githubusercontent.com/loblaw-sre/backstage-plugin-gitlab/main/src/assets/Backstage_Gitlab_Pipeline_Information.png"  alt="Contributors Languages Pipeline Status"/>
-<img src="https://raw.githubusercontent.com/loblaw-sre/backstage-plugin-gitlab/main/src/assets/Backstage_Gitlab_Merge_Request_information.png"  alt="Merge Request Information"/>
-<img src="https://raw.githubusercontent.com/loblaw-sre/backstage-plugin-gitlab/main/src/assets/Backstage_Gitlab_Issue_Information.png"  alt="Issue Information"/>
+This is the backstage gitlab plugin.
```
Comment (JellyBellyDev) [inline README.md:9]:
```
<p align="center">
  <img src="https://avatars.githubusercontent.com/u/10090828?s=200&v=4" width="200px" alt="logo"/>
</p>
<h1 align="center">Backstage Plugin GitLab</h1>

![release workflow](https://img.shields.io/github/workflow/status/immobiliare/backstage-plugin-gitlab/Release)
[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier?style=flat-square)
[![semantic-release](https://img.shields.io/badge/%20%20%F0%9F%93%A6%F0%9F%9A%80-semantic--release-e10079.svg?style=flat-square)](https://github.com/semantic-release/semantic-release)
![npm (scoped)](https://img.shields.io/npm/v/@immobiliarelabs/backstage-plugin-gitlab?style=flat-square)
![license](https://img.shields.io/github/license/immobiliare/backstage-plugin-gitlab)

> [Backstage](https://backstage.io/) plugins to to interact with [https://gitlab.com/](GitLab)
```
WDYT?

File: README.md [hunk 11]
```diff
@@ -1,30 +1,15 @@
...
-[![](https://img.shields.io/github/languages/code-size/loblaw-sre/backstage-plugin-gitlab)](https://github.com/loblaw-sre/backstage-plugin-gitlab)
-[![Hits](https://hits.seeyoufarm.com/api/count/incr/badge.svg?url=https%3A%2F%2Fgithub.com%2Floblaw-sre%2Fbackstage-plugin-gitlab&count_bg=%2379C83D&title_bg=%23555555&icon=&icon_color=%23E7E7E7&title=Visitors&edge_flat=false)](https://hits.seeyoufarm.com)
-
-
-Website: [https://gitlab.com/](https://gitlab.com/)
-
-<img src="https://raw.githubusercontent.com/loblaw-sre/backstage-plugin-gitlab/main/src/assets/Backstage_Gitlab_Pipeline_Information.png"  alt="Contributors Languages Pipeline Status"/>
-<img src="https://raw.githubusercontent.com/loblaw-sre/backstage-plugin-gitlab/main/src/assets/Backstage_Gitlab_Merge_Request_information.png"  alt="Merge Request Information"/>
-<img src="https://raw.githubusercontent.com/loblaw-sre/backstage-plugin-gitlab/main/src/assets/Backstage_Gitlab_Issue_Information.png"  alt="Issue Information"/>
+> [Backstage](https://backstage.io/) plugins to to interact with [https://gitlab.com/](GitLab)
```
Comment (JellyBellyDev) [inline README.md:12]: double `to`

**Review by JellyBellyDev — APPROVED** [review]
LGTM 👍

**Pull request merged**